com.unity.render-pipelines.high-definition/Runtime/RenderPipeline/HDRenderPipeline.cs

```diff
@@ -23,72 +23,24 @@ namespace UnityEngine.Rendering.HighDefinition
     /// </summary>
     public partial class HDRenderPipeline : RenderPipeline
     {
-        #region Default Settings
-        internal static HDRenderPipelineAsset defaultAsset
-            => GraphicsSettings.renderPipelineAsset is HDRenderPipelineAsset hdrpAsset ? hdrpAsset : null;
+        #region Global Settings
+        internal static HDRenderPipelineGlobalSettings defaultAsset
+            => HDRenderPipelineGlobalSettings.instance;
+
+        private HDRenderPipelineGlobalSettings m_globalSettings;
```

Comment:
I think it should be named "m_GlobalSettings" according to convention.

Reply:
fixed in next subsequent commit